.travis.yml

```diff
@@ -0,0 +1,30 @@
+language: rust
+cache: cargo
+rust:
+    - 1.35.0
```

Comment:
I think it's fine to aim for „current stable“, this is not a library with some stability promises. Let's just drop this one.

Reply:
Good advice, addressed in fcc2d6634d80879785c7027e478d77f70f8db915